src/mus2midi.c

```diff
@@ -0,0 +1,224 @@
+/*
+ * rv32emu is freely redistributable under the MIT License. See the file
+ * "LICENSE" for information on usage and redistribution of this file.
+ */
+
+/*
+ * Reference: https://github.com/fawtytoo/Mus2Midi
+ *
+ */
+
+#include <stddef.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+
+typedef struct {
+    char id[4];
+    uint16_t score_len;
+    uint16_t score_start;
+} __attribute__((packed)) hdr_mus_t;
+
+typedef struct {
+    char id[4];
+    int length;
+    uint16_t type;
+    uint16_t ntracks;
+    uint16_t ticks;
+} __attribute__((packed)) hdr_mid_t;
+
+static char magic_mus[4] = {
```

Comment:
It is a bit confusing to specify abbreviations such as `mus`, `mid`, `trk`, etc. Use the fully qualified ones.

Reply:
Enforce `static const` qualifiers to objects with prefix `magic_`.